playbooks/provisioning/openstack/README.md

```diff
@@ -41,12 +41,23 @@ etc.). The result is an environment ready for openshift-ansible.
 Pay special attention to the values in the first paragraph -- these
 will depend on your OpenStack environment.
 
-The `env_id` and `openstack_dns_domain` will form the DNS domain all
+The `env_id` and `public_dns_domain` will form the cluster's DNS domain all
 your servers will be under. With the default values, this will be
-`openshift.example.com`.
-
-`openstack_nameservers` is a list of DNS servers accessible from all
-the created Nova servers. These will be serve as your DNS forwarders.
+`openshift.example.com`. For workloads, the default subdomain is 'apps'.
+
+The `public_dns_nameservers` is a list of DNS servers accessible from all
+the created Nova servers. These will be serving as your DNS forwarders for
+external FQDNs that do not belong to the cluster's DNS domain.
+
+The `openshift_use_dnsmasq` controls either dnsmasq is deployed or not.
+When dnsmasq is not used, you may only effectively configure a 2nd and a 3rd
+"fallback" nameservers of the maximum of a 3 nameserver records allowed for
```

Comment:
Keep in mind that if nameserver 1 responds, none of the other nameservers will be used, so not sure why more than 3 is needed anyway. (we typically run with just 2 in the list).

Typo below `reoslv`...